tests/components/nws_alerts/test_config_flow.py

```diff
@@ -0,0 +1,6 @@
+"""Test the NWS Alerts config flow."""
```

Comment:
Make a search for `requests_mock` in the `tests/components` folder, or search for `respx` if you have gone async.